apps/sms/views/conversation/js/information.js

```diff
@@ -339,6 +346,42 @@ var VIEWS = {
           l10nContainsDateSetup(this.receivedTimestamp, message.timestamp);
           l10nContainsDateSetup(this.sentTimestamp, message.sentTimestamp);
           setL10nAttributes(this.sentTitle, 'message-sent');
+
+          var units = {
+            months: MONTH,
+            days: DAY,
+            hours: HOUR,
+            minutes: MINUTE,
+            seconds: SECOND
+          };
+          var lateness = message.timestamp - message.sentTimestamp;
+
+          this.latenessBlock.classList.remove('hide');
+          if (
+            !message.timestamp || 
+            !message.sentTimestamp || 
+            lateness < LATENESS_DISPLAY_LOWER_LIMIT
+          ) {
+            this.latenessBlock.classList.add('hide');
+          } else {
+            var rest = 0;
+            ['months', 'days', 'hours', 'minutes', 'seconds'].forEach((val) => {
+              var output = 0;
+              var camelCaseId = Utils.camelCase('lateness-' + val);
+              if(lateness >= units[val]) {
+                rest = lateness % units[val];
+                output = (lateness - rest) / units[val];
+                lateness = rest;
+              }
+
+              setL10nAttributes(
+                this[camelCaseId],
```

Comment:
in that case I think it's better if we just query it here, because it's used only here. What do you think ?